src/main/resources/i18n/Messages_en.properties

```diff
@@ -242,6 +242,19 @@ notification.seedBank.unknownAutomationTriggered.app.title={0} triggered at {1}
 notification.seedBank.unknownAutomationTriggered.email.body.1=Automation {0} has been triggered at {1}.
 notification.seedBank.unknownAutomationTriggered.email.body.2=Additional details\: {0}
 notification.seedBank.unknownAutomationTriggered.email.subject={0} triggered at {1}
+notification.support.requestSubmitted.email.body.1=Thank you for contacting Terraware support.
+notification.support.requestSubmitted.email.body.2=We rely on our users to help make Terraware better and appreciate hearing from you.
+notification.support.requestSubmitted.email.body.3=Within the next few days, we may reach out to obtain additional information that would help us investigate the issue or to provide you with the status of a bug fix.
+notification.support.requestSubmitted.email.body.4=In the meantime, you may find helpful information in the [Terraware - Knowledge Base].
```

Comment:
The hyphen in the link here looks a little weird to me; isn't it just the "Terraware Knowledge Base"?

Reply:
This is from Zapier triggers for now but let me check with PMs.